src/QueryBuilder.js

```diff
@@ -1566,6 +1570,229 @@ QueryBuilder.prototype.returning = queryMethod('returning');
 QueryBuilder.prototype.truncate = queryMethod('truncate');
 
 /**
+ * Json query APIs
+ */
+
+/**
+ * Json field filter.
+ *
+ * Supports having field expression in both sides of operand, but right hand value cannot
+ * be any other value, but only string reference to other column/json filed, json array
```

Comment:
Typo: "filed" --> "field". Also "be any other value, but only string reference to" could be clearer like this "be any other value than string reference to".